GiniVision.podspec

```diff
@@ -15,11 +15,24 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.author           = { 'Peter Pult' => 'p.pult@gini.net' }
   s.source           = { :git => 'https://github.com/gini/gini-vision-lib-ios.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/gini'
-
   s.ios.deployment_target = '8.0'
+  s.default_subspec = 'Core'
```

Comment:
I know this is what you saw in the blog post, but it's not what Cocoapods recommends. Read [this](https://guides.cocoapods.org/syntax/podspec.html#default_subspecs).

I think it's not so important and I'd say it's justified in your case, but you might want to think again. For this library, I guess you'd have to think if most people would use the core only, or networking as well.

For the pinning, I think it still shouldn't be a default.

Reply:
Exactly. I didn't make a decision yet because my first thought was what you say, having the whole package as default and then a subspec only with the UI and another with the pinning.